common/src/main/java/com/wynntils/features/tooltips/ItemStatInfoFeature.java

```diff
@@ -132,7 +133,7 @@ public void onTooltipPre(ItemTooltipRenderEvent.Pre event) {
         }
     }
 
-    private void updateItemName(GearInfo gearInfo, GearInstance gearInstance, LinkedList<Component> tooltips) {
+    private void updateItemName(GearInfo gearInfo, GearInstance gearInstance, Deque<Component> tooltips) {
```

Comment:
I know IntelliJ keeps suggesting this, but I disagree. In my view, Deque is not a commonly used interface in Java. In short, it was introduced too late. So when we need to have a double-ended list, I prefer to specificially request a LinkedList. It's not that we ever have any other implementation of Deque anyway.